engine/src/main/resources/assets/shaders/combine_frag.glsl

```diff
@@ -45,10 +45,10 @@ uniform sampler2D texEdges;
 uniform float outlineDepthThreshold;
 uniform float outlineThickness;
 
-# define OUTLINE_COLOR 0.0, 0.0, 0.0
+#define OUTLINE_COLOR 0.0, 0.0, 0.0
 #endif
 
-#if defined (VOLUMETRIC_FOG)
+#ifdef VOLUMETRIC_FOG
```

Comment:
I just read about the difference between #ifdef and #if defined. Good idea changing it.